spinegeneric/cli/params_checker.py

```diff
@@ -54,113 +53,101 @@ def main():
         level=logging.DEBUG,
     )
 
-    # Initialize the layout
-    with importlib.resources.path(
-        spinegeneric.config, "bids_specs.json"
-    ) as path_sg_layout_config:
+    # Initialize the BIDSLayout object directed at the input dataset.
+    # From the BIDS documentation:
+    #   "A BIDSLayout instance is a lightweight container for all files in the BIDS project directory."
+    with importlib.resources.path(spinegeneric.config, "bids_specs.json") as path_sg_layout_config:
+        # TODO: This step takes quite a long time, but nothing gets logged during. Maybe we could provide some feedback?
         layout = BIDSLayout(
             data_path,
-            indexer=BIDSLayoutIndexer(config_filename=path_sg_layout_config),
+            # BIDSLayoutIndexer is a class that indexes files based on pattern-matching defined in the config.
+            # By default, BIDS has its own config. But, SG specifies its own custom config instead. (Why?)
```

Comment:
I tracked it down:
- The `bids_specs.json` file was introduced in #198, specifically because at the time the default config file didn't support the `MTS` suffix in filenames. Thankfully the corresponding issue #190 links to the then-current upstream config file.
- Comparing `bids_specs.json` with the historical upstream config, only two lines are different, inside `default_path_patterns`, and they just add `MTS` as valid suffixes.
- Since [pybids#840](https://github.com/bids-standard/pybids/pull/840) from last year, upstream now supports the `MTS` suffix by default, but with a difference: it looks like `flip` and `mt` are mandatory entities in the filename when the suffix is `MTS`.

I think the upshot is that we can get rid of the custom config, and of the old handling of `MTS` (which relied on the `acq` entity, instead of `flip` and `mt`). I'll write a separate PR for that.

Reply:
Wonderful sleuthing!! :hearts: